cleanlab_studio/internal/api/api.py

```diff
@@ -379,3 +380,55 @@ def get_prediction_status(api_key: str, query_id: str) -> Dict[str, str]:
     handle_api_error(res)
 
     return cast(Dict[str, str], res.json())
+
+
+def tlm_prompt(
+    api_key: str,
+    prompt: str,
+    quality_preset: str,
+) -> JSONDict:
+    """
+    Prompt Trustworthy LLM with a question, and get back its answer along with a confidence score
```

Comment:
branding: "Trustworthy Language Model" or "TLM", not "Trustworthy LLM" (want to be consistent)